models/breast_density_classification/configs/inference.json

```diff
@@ -91,7 +92,14 @@
         },
         {
             "_target_": "StatsHandler",
-            "iteration_log": true
+            "iteration_log": false,
+            "output_transform": "$lambda x: None"
+        },
+        {
+            "_target_": "ClassificationSaver",
```

Comment:
Hi @vikashg , I pushed a new commit (https://github.com/Project-MONAI/model-zoo/pull/150/commits/74c44e34e4f14efa4c4d765d570024ad670cda97), and the `ClassificationSaver` is used to save the predictions into a csv file, you can check `monai.handlers.CheckpointSaver` for more details.

Reply:
Awesome Thanks. Now I can see the saved csv file.